robottelo/cli/base.py

```diff
@@ -431,7 +433,7 @@ def set_parameter(cls, options=None):
         return result
 
     @classmethod
-    def update(cls, options=None):
+    def update(cls, options=None, user=None, password=None):
```

Comment:
@rochacbruno @omaciel @sghai @oshtaier 

I was not aware of this. I can use it, but it seems to me as a hack. Context Managers purpose is to ensure something is closed properly and the only purpose of this code is pass "user" and "password" as parameters to ssh. Passing the parameteres itself as class attributes its very strange and when directly done, as on the old version of my_account.py can lead us to race conditions, once we are messing with "global variables".

I think we need to rethink the way CLI works because of all this bad smells. Having the parameters available with default values would be simpler. The user must only read the docs to see what they do instead of having to wait to a more experience tell him about this context manager.

I can do this solution right now but would like to rethink hammer architecture. What you guys think?